### PR TITLE
OCPBUGS-2837: Reduce debug logs in agent wait-for commands

### PR DESCRIPTION
**Pull request by zaneb**

### CODE REVIEW

File: pkg/agent/kube.go [hunk 1]
```diff
@@ -72,7 +73,7 @@ func (kube *ClusterKubeAPIClient) IsBootstrapConfigMapComplete() (bool, error) {
 	// Get latest version of bootstrap configmap
 	bootstrap, err := kube.Client.CoreV1().ConfigMaps("kube-system").Get(kube.ctx, "bootstrap", v1.GetOptions{})
 
-	if err != nil {
+	if err != nil && !k8serrors.IsNotFound(err) {
```
Comment (andfasano) [inline pkg/agent/kube.go:76]:
```suggestion
	if !k8serrors.IsNotFound(err) {
```
nit: `IsNotFound` should return false if error is nil

Comment (zaneb) [reply pkg/agent/kube.go:76]: It should, but we want this block to _not_ execute if the error is nil.

**Pull request merged**